distribution/lib/Standard/Base/0.0.0-dev/src/Meta.enso

```diff
@@ -582,26 +457,7 @@ get_simple_type_name value = @Builtin_Method "Meta.get_simple_type_name"
    Arguments:
    - value: the value to get the type of.
 get_qualified_type_name : Any -> Text
-get_qualified_type_name value = @Builtin_Method "Meta.get_qualified_type_name"
-
-## PRIVATE
-   Returns a short name of a type (the last part of its qualified name).
-
-   Arguments:
-   - typ: the type to get the short name of.
-get_short_type_name : Any -> Text
-get_short_type_name typ = @Builtin_Method "Meta.get_short_type_name"
-
-## PRIVATE
-   Returns the type that this constructor is a part of.
-
-   Arguments:
-   - constructor: the constructor to get the declaring type of.
-get_constructor_declaring_type : Any -> Any
-get_constructor_declaring_type constructor = @Builtin_Method "Meta.get_constructor_declaring_type"
-
-## PRIVATE
-instrumentor_builtin op args = @Builtin_Method "Meta.instrumentor_builtin"
+get_qualified_type_name value = @Tail_Call Meta_Helpers.get_qualified_type_name_builtin value
```

Comment:
This should be `(Meta.meta value : Meta.Type) . qualified_name`. There is quite a few usages, but I can rewrite them if we think it is a good idea?

Reply:
Given there are so many usages I think it may be good to have one common method that encapsulates this common pattern instead of reproducing the 'assert type + get qualified_name' in every place.

Reply:
It turns out that `Meta.get_qualified_type_name` works not only on `Type` but also on values. As such the replacement via `Meta.Type.qualified_name` isn't really possible. Only if I added `qualified_name` to all `Meta` types.

Easier to just keep the module static method.